dkandalintsev/login.feature

```diff
@@ -0,0 +1,136 @@
+    And user filled in the password field with invalid <password>
+    When user clicks "Next"
+    Then user sees that password input is red
+    And user has "Wrong password. Try again or click ‘Forgot password’ to reset it." message uder the password input
+
+Scenario: 1.08 User tries to sign in without saved Google account
+    Given user does not have saved Google accounts in browser
+    When user clicks "Sign in with Google" button
+    Then user sees "Email or phone" input for Google account
+
+Background:
+    Given user does not have saved Google accounts in browser
+    And user clicks "Sign in with Google" button
+
+Scenario: 1.09 User tries to sign in with non-existing email
+    Given user sees "Email or phone" input
+    When user fills in "Email or phone" input with <non-existing email>
+    Examples:
+        | non-existing email |
+        | ifmnvme@gmail.com  |
+    And user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Couldn't find your Google account" message under the input
+
+Scenario: 1.10 User tries to sign in with non-existing phone number
+    Given user sees "Email or phone" input
+    When user fills in "Email or phone" input with <non-existing phone>
+    Examples:
+        | non-existing phone |
+        | +79819139699       |
+    And user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Couldn't find your Google account. Try using your email address instead" message under the input
+
+Scenario: 1.11 User tries to sign in with incorrect password
+    Given user fills in "Email or phone" input with <email address>
+    And user clicks "Next"
+    When user fills in "Enter your password" input with <incorrect password>
+    Examples:
+        | incorrect password |
+        | Trewqasdf4321?     |
+    And user clicks "Next"
+    Then password input is outlined with red color
+    And user sees "Wrong password. Try again or click ‘Forgot password’ to reset it." message under the input
+
+Scenario: 1.12 User signs in with correct email and password
+    Given user fills in "Email or phone" input with <email address>
+    And user clicks "Next"
+    And user fills in "Enter your password" input with <password>
+    When user clicks "Next"
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.13 User does not fill email input
+    Given user leaves "Email or phone" input empty
+    When user clicks "Next"
+    Then email input is outlined with red color
+    And user sees "Enter an email or phone number" message under the input
+
+Scenario: 1.14 User does not fill password input
+    Given user fills in "Email or phone" input with <email address>
+    And user clicks "Next"
+    And user leaves "Enter your password" input empty
+    When user clicks "Next"
+    Then password input is outlined with red color
+    And user sees "Enter a password" message under the input
+
+Scenario: 1.15 User tries to log in with invalid data
```

Comment:
I'd like to suggest  another name for this case.
For example, "user tries to log-in with another domain"

Reply:
Probably you are right